eZ/Publish/Core/Persistence/Legacy/Content/Gateway/ExceptionConversion.php

```diff
@@ -704,14 +437,7 @@ public function insertRelation(RelationCreateStruct $struct)
         }
     }
 
-    /**
-     * Returns all Content IDs for a given $contentTypeId.
-     *
-     * @param int $contentTypeId
-     *
-     * @return int[]
```

Comment:
this could be useful if it's true

Reply:
Yes, this is why it's documented on parent `\eZ\Publish\Core\Persistence\Legacy\Content\Gateway::getContentIdsByContentTypeId`. Having it on child is redundant. 
Do you see now that we need `@inheritdoc`? :stuck_out_tongue_winking_eye: 
I can add it if you want. For some reason param and return doc is not inherited w/o it. At least in PhpStorm.